docs/ops/normalization/MVN_6.md

```diff
@@ -0,0 +1,91 @@
+## MVN <a name="MVN"></a> {#openvino_docs_ops_normalization_MVN_6}
+
+**Versioned name**: *MVN-6*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates mean-variance normalization of the input tensor.
+
+**Detailed description**
+
+*MVN* subtracts mean value from the input blob:
+\f[
+o_{i} = i_{i} - \frac{\sum{i_{k}}}{C * H * W}
+\f]
+If *normalize_variance* is set to 1, the output blob is divided by variance:
```

Comment:
Please, provide different formulas for different values of `how_eps_applied` parameter.